### PR TITLE
2.7 Docs Review

### PR DESCRIPTION
**Pull request by jamesnocentini**
[Mercury Documentation Plan](https://hub.internal.couchbase.com/confluence/display/techpubs/Mercury+Documentation+Plan)

### CODE REVIEW

File: modules/ROOT/examples/java-platform/src/com/couchbase/code_snippets/Examples.java [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+        mutableDoc.setString("language", "Java");
+        database.save(mutableDoc);
+
+        Document document = database.getDocument(mutableDoc.getId());
+        // Log the document ID (generated by the database) and properties
+        System.out.println("Document ID is :: " + document.getId());
+        System.out.println("Learning " + document.getString("language"));
+
+        // Create a query to fetch documents of type SDK.
+        Query query = QueryBuilder.select(SelectResult.all())
```
Comment (ibsoln) [inline modules/ROOT/examples/java-platform/src/com/couchbase/code_snippets/Examples.java]:
```suggestion :: 

The following query will list the existing documents; enabling easy review of server changes being replicated to client

`        Query query2 = QueryBuilder.select(SelectResult.expression(Meta.id),
                        SelectResult.property("language"),
                        SelectResult.property("version"))
                .from(DataSource.database(database))
                .where(Expression.property("type").equalTo(Expression.string("SDK")))
                .orderBy(Ordering.expression(Meta.id));
        try {
            ResultSet rs2 = query2.execute();
            for (Result result : rs2) {
                System.out.println("Student ID :: " + result.getString("id"));
                System.out.println("Learning :: " + result.getString("language") + " version :: "+ result.getFloat("version"));
            }
        } catch (CouchbaseLiteException e) {
            System.out.println(e.getLocalizedMessage());
        }
`

 ```

**Review by ibsoln — COMMENTED** [review]
test

File: modules/ROOT/examples/java-platform/src/com/couchbase/code_snippets/Main.java [hunk 2]
```diff
@@ -0,0 +1,8 @@
+package com.couchbase.code_snippets;
+
+public class Main {
+
+    public static void main(String[] args) {
+	// write your code here
+    }
+}
```
Comment (ibsoln) [inline modules/ROOT/examples/java-platform/src/com/couchbase/code_snippets/Main.java]: @ibsoln TODO?

**Review by ibsoln — CHANGES_REQUESTED** [review]
I don't understand this PR#133. It seems to be pushing 2.7 changes back into Release 2.6 and I don't believe that is right? 

Can you look at this for me please? @RichardSmedley

**Comment by RichardSmedley** [comment]
Looks like James made this against 2.6 in error - easily done in the GH ui.
I'm closing it.

**Comment by ibsoln** [comment]
Thanks Richard

**Pull request closed without merging**